cmd/collectors/rest/plugins/volume/volume.go

```diff
@@ -96,6 +114,73 @@ func (my *Volume) updateVolumeLabels(data *matrix.Matrix) {
 	}
 }
 
+func (my *Volume) handleARWProtection(data *matrix.Matrix) {
+	var (
+		arwInstance       *matrix.Instance
+		arwStartTimeValue time.Time
+		err               error
+	)
+
+	// Purge and reset data
+	my.arw.PurgeInstances()
+	my.arw.Reset()
+
+	// Set all global labels
+	my.arw.SetGlobalLabels(data.GetGlobalLabels())
+
+	learningMode := false
```

Comment:
comment might help explain where a break is needed or not

Reply:
handled